src/controllers/movie.js

```diff
@@ -0,0 +1,167 @@
+const { Prisma } = require('@prisma/client');
+const prisma = require('../utils/prisma');
+const getErorCode = (e, res) => {
+    if (e instanceof Prisma.PrismaClientKnownRequestError) {
+        if (e.code === "P2002") {
+            return res.status(409).json({ error: "A movie with the provided field already exists" })
+        } else if (e.code === "P2025") {
+            return res.status(404).json({ error: "A movie with the provided id does not exists!" })
+        }
+    }
+    res.status(500).json({ error: e.code + " " + e.message })
+}
+const getAllMovies = async (req, res) => {
+
+    const { runtimeLt, runtimeGt } = req.query;
+    let query = { include: { screenings: true } };
+
+    if (runtimeLt && runtimeGt) {
+        query = {
+            where: {
+                runtimeMins: {
+                    gte: Number(runtimeGt),
+                    lte: Number(runtimeLt)
+                }
+            },
+            include: { screenings: true }
+        }
+    }
```

Comment:
What if the client wants to get movies with a runtime greater than 30 minutes but they don't care about less than?